src/model/utils.js

```diff
@@ -69,3 +69,33 @@ export function generateMetaGraph(metaNodes) {
     }
     return metaGraph
 }
+
+
+export function findTopLeftCorner(ldraw, pos) {
+    let minX = Infinity;
+    let minY = Infinity;
+    let maxX = -Infinity;
+    let maxY = -Infinity;
+
+    ldraw.forEach(element => {
+        if (element.op.toUpperCase() === 'P') {
```

Comment:
Additionally could you let me know how are we getting the xdot notation?
I noticed that sometimes it contains 'p' in the ldraw operation and other times 'P'. I wanted to check if they are the same